dev-libs/processor-trace/processor-trace-1.6.1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
To be honest, I don't like the idea of changing copyright on ebuild that is otherwise untouched. Given that it's not going to stay around for long, let's not change it.

Reply:
Fixed.